include/alpaka/acc/AccCpuOmp2Blocks.hpp

```diff
@@ -151,7 +151,9 @@ namespace alpaka
                         // m_sharedMemSizeBytes
                         static_cast<size_t>(AccCpuOmp2Blocks<TDim, TIdx>::staticAllocBytes()),
                         // m_globalMemSizeBytes
-                        getMemBytes(dev)};
+                        getMemBytes(dev),
+                        //m_cooperativeLaunch
+                        std::false_type()};
```

Comment:
shouldn't this be simply
```suggestion
                        false};
```
?